README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
- [ ] Please update the react default README file with a more-related one using this great [template](https://github.com/microverseinc/readme-template) from Microverse